src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md

```diff
@@ -0,0 +1,68 @@
+
+Recommendations
+
+Reduce Supply and Borrow Caps on Aave V3, Optimism: Given that the market’s size has decreased, as well as a decrease in on-chain supply — from 31M pre-freeze to 23.6M now — we recommend decreasing the supply and borrow caps ahead of unfreezing the asset. This will ensure that any future growth in the market is controlled and risk is minimized.
+
+We recommend setting the caps according to our initial listing methodology, in which we determine the amount of an asset that can be liquidated with slippage lower than the LB, set at 1% in the stablecoin E-mode, and recommend the supply cap be double that amount. This leads us to a recommendation of a 7M sUSD supply cap and a borrow cap set at 80% of the supply cap at 5.6M sUSD.
+Reduce Stablecoin E-Mode LT and LTV: We recommend reducing E-Mode LT and LTV for stablecoins in Optimism’s Stablecoin E-Mode. Reducing LTs will make the protocol more resilient to any potential depegs in the future, ensuring that there is larger buffer in the event a stablecoin’s price falls and liquidations occur. Additionally, lowering LTV will slightly reduce users’ stablecoin-stablecoin borrowing power, again reducing risk in these markets.
+
+A 2% LT reduction on the Optimism V3 stablecoin E-mode category will make 18 accounts eligible for liquidation, potentially inducing $25K in new liquidations.
+Adjust sUSD’s LTV and LT: Following the freeze of sUSD, its non-E-Mode LTV was dropped to 0%. Its current LT is 75%. Increasing the LTV will allow users to borrow non-stablecoin assets against their collateral, while adjusting the LT downwards will ensure the protocol has a larger buffer in the event of liquidations, whether from a depeg or from appreciation of borrowed assets.
+
+We recommend increasing sUSD’s non-E-Mode LTV to 60% and reducing its non-E-Mode LT to 70%. At this time, this would lead to a negligible $8 in liquidation across 4 accounts.
+Unfreeze the sUSD market on Optimism: While sUSD has repegged, we note that the ongoing deprecation of synth assets may continue to cause peg volatility. The discount rate, which dictates the value synth asset holders can receive when they redeem, will continue to be lowered in an effort to deprecate these markets. Synth asset holders may request a redemption at parity, which, in the future, may carry some time-based conditions (i.e., sUSD is sent to the redeeming user over the course of weeks or months) to reduce the potential impact on sUSD’s peg. While the Synthetix Council has an incentive to ensure redemptions of synth assets with minimal price impacts, whether to the synth assets or sUSD, these time-based conditions are not codified.
```

Comment:
here i see the recommendation to unfreeze the asset, but its not implemented

Reply:
@rustboyar Added the unfreeze of sUSD in this commit - https://github.com/bgd-labs/aave-proposals-v3/pull/346/commits/66d6d970f6ed8ddb43d44b790320ba2c8c68ff10